### PR TITLE
ACMS-1711: do not import media if installed from profile.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #ACMS-1711

**Proposed changes**
do not import media if installed from profile.

**Alternatives considered**
NA

**Testing steps**
- Checkout ACMS-1711 branch using `git clone --branch ACMS-1711 git@github.com:acquia/acquia_cms.git`
- Install all dependencies using `composer install && composer update --lock`
- Install site using `./vendor/bin/drush si minimal --account-pass=admin && drush in acquia_cms_image -y`
- Verify image media created at `/admin/content/media`. 
- Export config using `./vendor/bin/drush cex -y`
- Get site uuid using `./vendor/bin/drush cget system.site uuid`
- Copy the uuid
- **Site install then config import:**
- Install site using `./vendor/bin/drush si minimal --account-pass=admin && ./vendor/bin/drush cset system.site uuid <copied-uuid> -y && ./vendor/bin/drush cim -y && ./vendor/bin/drush cr`
- Verify image media created at `/admin/content/media`. 
- **Site install with existing site:**
- Install site using `./vendor/bin/drush si minimal --account-pass=admin --existing-config && drush cr`
- Verify image media created at `/admin/content/media`. 

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_image/acquia_cms_image.install [hunk 1]
```diff
@@ -19,6 +19,12 @@ function acquia_cms_image_install($is_syncing) {
   if (!$is_syncing) {
     _acquia_cms_common_editor_config_rewrite(TRUE);
 
+    // If installed from profile then return.
+    global $install_state;
+    if (isset($install_state['active_task']) && $install_state['active_task'] == "install_profile_modules") {
+      return;
+    }
+
```
Comment (chandan-singh7929) [inline modules/acquia_cms_image/acquia_cms_image.install:27]: Also verify that it works on site install with existing configurations.

Comment (rajeshreeputra) [reply modules/acquia_cms_image/acquia_cms_image.install:27]: Yes, it works with:
- `./vendor/bin/drush si minimal && ./vendor/bin/drush  in acquia_cms_image -y && ./vendor/bin/drush cex`
- `./vendor/bin/drush si minimal --existing-config`

File: modules/acquia_cms_image/acquia_cms_image.install [hunk 2]
```diff
@@ -19,6 +19,12 @@ function acquia_cms_image_install($is_syncing) {
   if (!$is_syncing) {
     _acquia_cms_common_editor_config_rewrite(TRUE);
 
+    // If installed from profile then return.
```
[the remark this answers is not in the document]
Comment (rajeshreeputra) [reply modules/acquia_cms_image/acquia_cms_image.install:22]: sure, updated

**Review by chandan-singh7929 — APPROVED** [review]
LGTM

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**